### ISSUE TITLE
Changes not reflected any more  (adding, removing entities from a room)

### ISSUE THREAD

**Issue by diothar** [dwainscheeren/dwains-lovelace-dashboard#407]
Hi, I had this happen when I first started using Dwain's and then deleted everything and started from scratch.  Seemed easier to start from scratch when I didn't have much working yet. But it came back. 

I noticed that I had a wrong area card showing up in a room, so I tried to fix it.  No matter what, I couldn't get it to stop showing up in my room.  Then I decided to move an entity to another room, and it did not reflect this at all. I tried adding a new room, the new room doesn't show up.  So there's some sort of condition that does pass validation when you reload the theme, but that it will not reflect any of the changes you make.  I've tried clearing caches, a separate private window, and a different computer with different browser.  I go back and look at the config files and the changes I made do show as expected.  But it seems to be possible to get the dashboard into a state where you save it, don't get errors, but it refuses to reflect changes.  

No changes to any rooms or entities are now being reflected.  And I don't know exactly where to start troubleshooting that.  No standout errors in my developer tools on my browser, either.

**Comment by diothar** [comment]
[](url)Worked with Seechay on discord- here's some new details:

killed my whole rooms.yaml.  Tried your intro rooms.yaml, still didn't display. 
Killed my whole addons folder
Now used your default and it worked.
Reintroduced each room one at a time.
Found I had to remove all the addons from rooms.yaml as well (the two rooms that had them would kill it).

I need to figure out the addons, because they were working, but I spent several days trying to fix an issue only to realize that the error wasn't enough to cause an error when you reloaded, but didn't display the new content.  So I was "stuck" and "frozen in time" regardless of my changes, and that's probably not a good thing. 

attaching my rooms.yaml
attaching a screenshot, and the test was to move light.laundry into guest bathroom and anywhere but hallway and entrance.  Those were where it was, and moving it was not reflected.

**Comment by diothar** [comment]
![Capture](https://user-images.githubusercontent.com/94809600/155274698-cf3daa8f-7c81-4968-836d-0cc369241955.PNG)
[dwains.zip](https://github.com/dwainscheeren/dwains-lovelace-dashboard/files/8122468/dwains.zip)
the archive had a few things not working, so I am going to kill those various cards. but this way you could see what I was trying to fix, but never saw it reflected until I killed everything.

There was no indication that my rooms.yaml wasn't loaded or whatever is happening- but I would continuously try to tweak the code to fix my other errors and it took a long time to notice that the room.yaml wasn't being parsed.

**Comment by dwainscheeren** [comment]
When you got an error you can check your HA error log it shows the file and line where you made a mistake/ have an error.

**Comment by diothar** [comment]
But I didn't get an error.  The only symptom I had was that my dashboard wasn't updating, and I didn't catch that for a few days.  I kept thinking whatever I was doing to try to fix the issue wasn't fixing it, but it turns out there was something preventing the dashboard from loading new content.  But there were no errors when I saved and reloaded.

(Yes, I've had stuff mess up so badly that I got errors when I reloaded, so I know it's there.  But it seems there are certain "less serious" errors that absolutely need to be throwing errors.
